### PR TITLE
Angel's USERNAME-card

### PR DESCRIPTION
**Pull request by lexuanangel**
Add my card information to index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
...
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Angel Le</p>
+          <p class="contact">
+            <i class="fa-brands fa-facebook"></i>
+            <a href="https://www.facebook.com" target="_blank">https://www.facebook.com/angelxuanle</a>
```
Comment (malakada) [inline index.html:159]: Uh oh! I think you intended for this link to go to your Facebook, but what you put in the `href` attribute only points to `https://www.facebook.com`.

To fix this, you just need to move what you put inside of your `<a>` tag into that `href` attribute, and then you're free to put whatever display text you want into the middle! 😃 

Here's an example that would fix this link:

```
<a href="https://www.facebook.com/angelxuanle" target="_blank">My super awesome Facebook profile!</a>
```

File: index.html [hunk 2]
```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">https://github.com/lexuanangel</a>
```
Comment (malakada) [inline index.html:161]: 😅 You'll want to fix this link the same way you need to fix your Facebook link that I commented on above.

File: index.html [hunk 3]
```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ==================  TEMPLATE  ================== -->
```
Comment (malakada) [inline index.html:151]: Looks like you accidentally copied too many lines when you were making a copy of the template! 😅 

No worries, it happens to the best of us. Just remove this line and the one below it and you'll be set. 💪

**Review by malakada — CHANGES_REQUESTED** [review]
Thanks so much for the contribution @lexuanangel! 😄 

The structure and content looks good, but there are a few links that need to be fixed. You also accidentally copied some extra comments that you didn't need to when you pasted in the template. Can you be sure to remove them so that future contributors (like yourself!) aren't confused by them? 😅 

I made comments inline in your code to help you. 😌 Can't wait to see you get this PR merged once you make the fixes! 😄🏆

File: index.html (hunk 1 above)
Comment (lexuanangel) [reply index.html]: I have made the change

File: index.html (hunk 3 above)
Comment (lexuanangel) [reply index.html]: I think I have made the change.

File: index.html (hunk 2 above)
Comment (lexuanangel) [reply index.html]: I have made this change.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@lexuanangel, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**